about.html

```diff
@@ -0,0 +1,135 @@
+      <ul>
+        <li><a href="#"><i class="fa fa-facebook" ></i></a></li>
+        <li><a href="#"><i class="fa fa-twitter" ></i></a></li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+    </div>
+
+    <nav>
+      <div class="logo">
+        <a href="index.html"><img src="./images/new-logo.png" alt="Logo"></a>
+      </div>
+      <div class="menu-items">
+        <ul class="nav-link">
+          <li class="link"><a href="#" id="aboutit">About</a></li>
+          <li class="link"><a href="#">Program</a></li>
+          <li class="link"><a href="#">Join</a></li>
+          <li class="link"><a href="#">Sponsers</a></li>
+          <li class="link"><a href="#">News</a></li>
+
+        </ul>
+        <div class="header-button"><button type="button" class="button-header">G Gathering</button></div>
+      </div>
+    </nav>
+  </section>
+  <div id="mySidenav" class="sidenav">
+    <a href="#" class="closebtn" onclick="closeNav()">&times;</a>
+    <a href="about.html">About</a>
+    <a href="#">Program</a>
+    <a href="#">Join</a>
+    <a href="#">Sponsers</a>
+    <a href="#">News</a>
+  </div>
+  <section class="about" id="about">
+    <div class="main">
+      <div class="hamburger" onclick="openNav()"><img src="./images/Ham.jpg" alt="hamburger"></div>
+
+      <div class="content">
+        <h1>"Welcome! to the Marketing world"</h1>
+        <h2>A GLOBAL GATHERING <br>OF BRAND BUILDERS & MARKETERS</h2>
+        <div class="inner-content">
+          <p>The Marketing Conference spotlights the latest innovations and breakthroughs in the marketing and advertising industry. Connect with the global community and join thought leaders and storytellers as they explore visionary ideas that engage and
+            inspire
+          </p>
+        </div>
+        <div class="inner-content-bottom">
+          <p>Please contact as per Email for any further questions about global Gathering</p>
+          <h3>globalgather@ggather.org</h3>
+          <div class="under-line"> </div>
+
+        </div>
+      </div>
+
+    </div>
+    <div class="featured-speakers">
+      <div>
+        <h2>About Marketing Conference That’s Bringing This Event To Life!</h2>
+        <div class="under-line"> </div>
+      </div>
+      <div>
+        <p>Marketing Conference is a premier marketing and advertising conference that will draw eminent leaders of the community from all over the world. <br><span class="content-hide">it aspires to help businesses connect with their customers better, enhance their visibility, and build their brand by learning from the best marketers. <br> The conference will unveil the latest technologies, research, and ideas that sell in the realms of content creation, branding, marketing, and advertising.industry. </span></p>
+      </div>
+      <div class="about-content-logo"><img src="./images/new-logo.png" alt="about logo" /></div>
+
+
+    </div>
```

Comment:
Kindly try to match the `Behance` design for the logo section in the mobile version on the about page. the `font size` needs to be smaller.
| Your design | Behance design |
| --- | --- |
|  ![screencapture-raw-githack-monika-soam-Marketing-Conference-About-Page-about-html-2022-03-10-00_04_53 - Copy](https://user-images.githubusercontent.com/24703171/157547141-e353f423-dc68-4453-886f-3e1b0c3cd475.png) | ![image](https://user-images.githubusercontent.com/24703171/157546866-f3dc3b66-7ccd-46ed-8556-ad912a4e94ab.png) |

Reply:
This is fixed

Reply:
Its fixed